### PR TITLE
Add toLowerCase and toUpperCase methods to sanitizers

### PR DESCRIPTION
**Pull request by liqiang372**
related issue https://github.com/express-validator/express-validator/issues/581

### CODE REVIEW

File: src/chain/sanitizers.ts [hunk 1]
```diff
@@ -18,4 +18,6 @@ export interface Sanitizers<Return> {
   toInt(radix?: number): Return;
   trim(chars?: string): Return;
   whitelist(chars: string): Return;
+  toLowerCase(): Return;
+  toUpperCase(): Return;
```
Comment (gustavohenke) [inline src/chain/sanitizers.ts:22]: Can you please keep these in alphabetic order among others?

File: src/chain/sanitizers-impl.ts [hunk 2]
```diff
@@ -63,4 +63,10 @@ export class SanitizersImpl<Chain> implements Sanitizers<Chain> {
   whitelist(chars: string) {
     return this.addStandardSanitization(validator.whitelist, chars);
   }
+  toLowerCase() {
+    return this.customSanitizer(value => (typeof value === 'string' ? value.toLowerCase() : value));
+  }
+  toUpperCase() {
+    return this.customSanitizer(value => (typeof value === 'string' ? value.toUpperCase() : value));
+  }
```
Comment (gustavohenke) [inline src/chain/sanitizers-impl.ts:71]: ditto

Comment (liqiang372) [reply src/chain/sanitizers-impl.ts:65]: done

File: src/chain/sanitizers.ts (hunk 1 above)
Comment (liqiang372) [reply src/chain/sanitizers.ts:20]: done

**Comment by gustavohenke** [comment]
Out in [`v6.7.0`](https://github.com/express-validator/express-validator/releases/tag/v6.7.0)!

**Pull request merged**